packages/GA4Client/src/event-handler.js

```diff
@@ -3,11 +3,17 @@ function EventHandler(common) {
 }
 
 EventHandler.prototype.sendEventToGA4 = function (eventName, eventAttributes) {
-    gtag(
-        'event',
-        this.common.truncateEventName(eventName),
-        this.common.truncateEventAttributes(eventAttributes)
-    );
+    if (this.common.forwarderSettings.enableDataCleansing) {
```

Comment:
The issue is that it is not being sanitized already in line 19 in this file (regular events), it is only being sanitized in the commerce events. Additionally, the truncation is supposed to happen regardless of if `enableDataCleansing` is on or off because that existed beforehand.  I will DRY this up a bit for now.  I think once the other PR is in.  We can do 1 more final review of the code and see where code can be optimized.

Reply:
Because truncating has always been happening, and what we are introducing is standardizing, I think I will simplify this further. Originally Brandon moved the truncation to the standardizing logic. I will remove that for now to reduce the number of code changes.